routes/user_router.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const Router = require('express').Router;
+const User = require(__dirname + '/../models/user');
+const bodyParser = require('body-parser').json();
+const jwToken = require(__dirname + '/../lib/jwt_auth');
+
+let userRouter = module.exports = Router();
+
+userRouter.route('/user')
+
+.post(jwToken, bodyParser, (req, res) => {
+  let newUser = new User(req.body);
+  newUser.save((err, data) => {
+    if (err) {
+      res.send(err);
+    }
+    res.status(200).json(data);
+  })
```

Comment:
this is closing out the .save function call, so need another curly bracket and parens to close out the post route declaration